### PR TITLE
Item Fixes:

### PR DESCRIPTION
**Pull request by Jedzkie**
Issue #980

### CODE REVIEW

File: db/pre-re/item_db.conf [hunk 1]
```diff
@@ -22075,8 +22075,8 @@ item_db: (
 	Loc: 256
 	View: 61
 	Script: <"
-		bonus bAgi,2;
-		bonus2 bSubEle,Ele_Ghost,10;
+		bonus bAgi, 2;
+		bonus2 bSubEle, Ele_Ghost, 10;
```
Comment (zackdreaver) [inline db/pre-re/item_db.conf]: it's 15% in kRO...
but 10% in fRO aegis

Comment (Jedzkie) [inline db/pre-re/item_db.conf]: Even in pre-re? Its 15%?

Comment (zackdreaver) [inline db/pre-re/item_db.conf]: pre-renewal kRO is hard to find this day,
I'll use twRO's pre-renewal instead.
And yes, it's 15%
ftp://ropatch.gameflier.com/Patch/2010-05-17itemdata_main.gpf

Comment (Jedzkie) [inline db/pre-re/item_db.conf]: Done. Thank you.

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1170)

**Pull request closed without merging**